api/server.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const helmet = require('helmet');
+
+const tasksRouter = require('./routers/taskRouter')
+const projectsRouter = require('./routers/projectRouter')
+const resourcesRouter = require('./routers/resourceRouter')
+
+const server = express();
+
+server.use(helmet())
+server.use(express.json())
+
```

Comment:
you can chain these in one server.use:

server.use( .  helmet(), express.json()  )